operator/v1/types_kubeapiserver.go

```diff
@@ -33,6 +33,23 @@ type KubeAPIServerSpec struct {
 
 type KubeAPIServerStatus struct {
 	StaticPodOperatorStatus `json:",inline"`
+
+	// serviceAccountIssuers tracks history of used service account issuers.
+	// The item without expiration time represents the currently used service account issuer.
+	// The other items represents service account issuers that were used previously and are still being trusted.
+	// see: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection
+	// +optional
+	ServiceAccountIssuers []ServiceAccountIssuerStatus `json:"serviceAccountIssuers,omitempty"`
+}
+
+type ServiceAccountIssuerStatus struct {
+	// name is the name of the service account issuer
+	Name string `json:"string"`
+
+	// expirationTime is the time after which this service account issuer will be pruned and removed from the trusted list
+	// of service account issuers.
```

Comment:
Please document what this means for the value to be omitted

Typically we avoid pointers in configuration APIs but that means that, for a time, you get a weird zero value so I think we are ok here

Reply:
There were already APIs with pointers for time values. We might consider using non-pointer and check against "default" value for time, but I found that more ugly :-)